### DISCUSSION TITLE
DNS issue - Vercel

### DISCUSSION THREAD

**Discussion by karkir0003** [vercel/community#5290] [Help] [answered]
### Summary

Hi team, I'm trying to deploy a dummy Next.js application to Vercel to get a feel for the tool. While I was able to click the buttons in the UI and follow the steps, the domain the dummy Next.js application was registered to was a long URL with a `.vercel.app`. I want to deploy the dummy application to a domain of `nextjs-dimension.com`. However, when I create the domain within the Vercel console and add the A record (b/c apex domain) yesterday and check again today morning, I see that the domain is marked with "invalid configuration". 

I followed these [instructions](https://vercel.com/docs/projects/domains/add-a-domain) to update the DNS. 

This is the repo that contains the dummy Nextjs application: [Here](https://github.com/karkir0003/nextjs-dimension-template)

### Example

https://vercel.com/karkir0003/nextjs-dimension-template/35z9J53Com6n5mXL8pk166kgw14U

### Steps to Reproduce

1. Working domain: [Here](https://nextjs-dimension-template-git-main-karkir0003.vercel.app/)
2. Domain I want to see working: [Here](https://www.next-dimension.com/)
3. 
![cursor](https://github.com/vercel/community/assets/54720987/541b4797-1e51-4c0a-a877-24979dc14be8)

**Comment by swarnava** [comment, marked as answer]
Hi,

It looks like your DNS `A` record hasn't propagated yet: https://dnschecker.org/#A/nextjs-dimension.com. Once it's setup fully, your domain should be up and running. You can contact to domain provider support team to investigate this further.

**Reply by karkir0003** [reply]
I see @swarnava. So, I did the steps correctly on my end? There's no specific support team at Vercel that can help me if I've waited say for a week and the custom domain DNS doesn't propagate.

**Reply by swarnava** [reply]
Hi,

> I did the steps correctly on my end?

We cannot confirm that since the actual domain is hosted in different location where you bought the domain.

> There's no specific support team at Vercel that can help me if I've waited say for a week and the custom domain DNS doesn't propagate.

Unfortunately no, since the domain isn't bought from us and we don't have visibility on your domain settings.

Are you sure you are the right owner? as I can see that nextjs-dimension.com is still available to purchase: 
<img width="348" alt="image" src="https://github.com/vercel/community/assets/1162991/3ceef2d8-de69-4a27-93ca-60ad703c6df9">

**Reply by karkir0003** [reply]
I think that clarifies the problem. I think I had to have bought the domain from Vercel page or some other site like GoDaddy, Namecheap, etc

**Reply by swarnava** [reply]
You can see the instruction here: https://vercel.com/docs/getting-started-with-vercel/buy-domain :)
